tests/src/core/physics/PhysicsScene.test.ts

```
@@ -537,6 +536,286 @@ describe("Physics Test", () => {
+          // not interfere.
+          const sweepCenter = new Vector3(3, 0, 0);
+          const halfExtents = new Vector3(0.5, 0.5, 0.5);
+          const direction = new Vector3(0, 1, 0);
+          const orientation = new Quaternion(0, 0, 0, 1);
+          nativeScene.boxCast(
+            sweepCenter,
+            orientation,
+            halfExtents,
+            direction,
+            100,
+            (sweepUuid: number) => {
```

ESLint

[error] Replace `⏎············sweepCenter,⏎············orientation,⏎············halfExtents,⏎············direction,⏎············100,⏎···········` with `sweepCenter,·orientation,·halfExtents,·direction,·100,`

(prettier/prettier)